python/GangaTest/Lib/GListApp/GListApp.py

```diff
@@ -18,13 +20,14 @@ class GListApp(IApplication):
     _name = 'GListApp'
     _schema = Schema(Version(1, 0), {
         'bound_print_comp': ComponentItem('files', defvalue=[], sequence=1, summary_print='_print_summary_bound_comp', typelist=['str', 'Ganga.test.GPI.GangaList.TFile.TFile']),
-        'bound_print_simple': SimpleItem(defvalue=[], sequence=1, summary_print='_print_summary_bound_simple'),
+        'bound_print_simple': SimpleItem(defvalue=GangaList(), summary_print='_print_summary_bound_simple'),
```

Comment:
I had problems in the past with doing this in several places without addressing some problems in the Schema, Is this object constructable from the interactive prompt? It could well be whatever was causing the problems I was seeing to go away.